platform/mellanox/docker-syncd-mlnx.mk

```diff
@@ -14,5 +14,8 @@ ifeq ($(SDK_FROM_SRC), y)
 $(DOCKER_SYNCD_BASE)_DBG_DEPENDS += $(MLNX_SDK_DBG_DEBS) $(MLNX_SAI_DBGSYM)
 endif
 
+$(DOCKER_SYNCD_BASE)_VERSION = 1.0.0
+$(DOCKER_SYNCD_BASE)_PACKAGE_NAME = syncd
```

Comment:
How do we distinguish syncd 1.0.0 built for various platforms, broadcom, mellanox, ...

Reply:
Stored in different repositories. Althoug there is no such question or problem till we actually make syncd a package.